docs/zh_cn/tutorial/03_pytorch2onnx.md

```diff
@@ -341,6 +341,6 @@ def _interpolate_helper(name, dim, interpolate_mode):
 
 1. Asinh 算子出现于第 9 个 ONNX 算子集。PyTorch 在 9 号版本的符号表文件中是怎样支持这个算子的？
 2. BitShift 算子出现于第11个 ONNX 算子集。PyTorch 在 11 号版本的符号表文件中是怎样支持这个算子的？
-3. 在\[第一篇教程\](./chapter_01_introduction_to_model_deployment.md）中，我们讲过 PyTorch （截至第 11 号算子集）不支持在插值中设置动态的放缩系数。这个系数对应 `torch.onnx.symbolic_helper._interpolate_helper`的symbolic_fn的Resize算子映射关系中的哪个参数？我们是如何修改这一参数的？
+3. 在 [第一篇教程](chapter_01_introduction_to_model_deployment.md) 中，我们讲过 PyTorch （截至第 11 号算子集）不支持在插值中设置动态的放缩系数。这个系数对应 `torch.onnx.symbolic_helper._interpolate_helper`的symbolic_fn的Resize算子映射关系中的哪个参数？我们是如何修改这一参数的？
```

Comment:
do not have `chapter_01_introduction_to_model_deployment.md`, should it be `01_introduction_to_model_deployment.md`?

Reply:
Done. Original doc missing  an "`" symbol,  so   `doc_link_checker.py`  skip the whole part after code block.